packages/semcom-core/lib/component/models/component-data.model.ts

```diff
@@ -0,0 +1,6 @@
+import { Quad } from 'rdf-js';
+
+/**
+ * Describes the kinds of data a component could work with.
+ */
+export type ComponentData = Quad[];
```

Comment:
Given our decision to only support quads in the new functions, I would strongly favor using that typing directly. Adding an alias has no real advantage, but it does make code harder to read.